docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/field-message/field-message.scss

```diff
@@ -87,7 +87,9 @@
           border-color: map.get($values, 'border-color');
 
           .civictheme-icon {
-            background-color: map.get($values, 'icon-color');
+            @if $theme == 'light' {
```

Comment:
Color only needed for light theme, For dark its inherited

Reply:
@joshua-salsadigital 
If it is inherited - create a variable with `inherit` or `false` value and add a condition here. please do not create customisations to generators - this does not align with other implementations we do